src/styles/bar.scss

```
@@ -0,0 +1,74 @@
+/* Header */
+.header {
+  display: flex;
+  flex-wrap: nowrap;
+  width: 100%;
+  background: $bcg-color;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  font-family: roboto, sans-serif;
+  font-size: 12px;
+  color: $clr-text;
+}
+
+.header_logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+/* Navigation */
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  align-items: center;
+}
+
+.nav__item {
+  text-transform: uppercase;
+  margin-right: 20px;
+  font-weight: 500;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  color: $clr-text;
+  white-space: nowrap;
+  height: 60px;
+  align-items: center;
+  padding: 20px 0; /* Вертикальні відступи */
+}
```

Use scss nesting everywhere
```suggestion

  :hover {
    color: $blue-color;
  }
}
```